src/config-local.tsx

```diff
@@ -43,6 +43,15 @@ export default {
 
   BASIC_CHANNEL_ID: 0,
   INCENTIVIZED_CHANNEL_ID: 1,
-  ACTIVE_CHANNEL: Channel.INCENTIVIZED
-
+  ACTIVE_CHANNEL: Channel.INCENTIVIZED,
+
+  PARACHAIN_LIST: [
+    { "parachainName":'Snowbridge', "parachainId": '0', 'isDisabled':false },
+    { "parachainName":'Snowbridge-test', "parachainId": '1001','isDisabled':false },
```

Comment:
I am new to typescript but would we not want to use number instead of string for paraId?